_rules/non-visual-reference-alternative-9bd38c.md

```diff
@@ -0,0 +1,407 @@
+- [F14: Failure of Success Criterion 1.3.3 due to identifying content only by its shape or location](https://www.w3.org/WAI/WCAG21/Techniques/failures/F14)
+- [F26: Failure of Success Criterion 1.3.3 due to using a graphical symbol alone to convey information](https://www.w3.org/WAI/WCAG21/Techniques/failures/F26)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The content in the second column is indicated with the word "right" (which is an indicator based on visual perception) but also indicated by referencing the word "howdy".
+
+```html
+<head>
+	<title>Passed example 1 9bd38c</title>
+	<style>
+	.col-container {
+		display: table;
+		width: 100%;
+	}
+	.col {
+		display: table-cell;
+		padding: 16px;
+	}
+	</style>
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Click "howdy" on the right, for a surprise</p>
+		</div>
+		<div class="col">
+			<button onclick="alert('Surprise!')">Howdy</button>
+		</div>
+	</div>
+</body>
+```
+
+#### Passed Example 2
+
+The button in the second column is indicated with the word "box" (which is a location indicator based on visual perception) but also indicated by referencing that the content can be found below this content in the DOM order. Note that "below" is also a visual reference word but in this case it can also be accurately interpreted as "next in the DOM order" which does not rely on visual attributes alone.
+
+```html
+<div class="col-container">
+  <div class="col">
+    <p>Interact with the box below this paragraph, for a surprise</p>
+  </div>
+
+  <div class="col">
+    <button onclick="alert('Surprise!')">Howdy</button>
+  </div>
+</div>
+```
+
+#### Passed Example 3
+
+The visual reference made by the word "right" is complemented by the non-visual reference made by the word "menu" to the content identified by the "Menu" heading.
+
+```html
+<head>
+	<title>Passed example 3 9bd38c</title>
+	<style>
+	.col-container {
+		display: table;
+		width: 100%;
+	}
+	.col {
+		display: table-cell;
+		padding: 16px;
+	}
+	</style>
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Find the menu on the right, to navigate</p>
+		</div>
+		<div class="col">
+			<h1>Menu</h1>
+			<ul>
+				<li>
+					<a href="https://www.w3.org/Consortium/contact">Contact</a>
+				</li>
+				<li>
+					<a href="https://www.w3.org/Help/">Help and FAQ</a>
+				</li>
+			</ul>
+					
+		</div>
+	</div>
+</body>
+```
+
+
+#### Passed Example 4
+
+This document is using the word "square" but in this case it is not an instruction about web content. In fact it is not even a reference of a shape in this case.
+
+```html
+	<p>In mathemathics a square is the result of multiplying a number by itself.</p>
+```
+
+#### Passed Example 5
+
+The following text is tilted and describes web content. But the block of content also includes this word "this" which makes it apparent that the description is about the same block of content.
```

Comment:
We shall make this part of an instruction.

<div class="tilt">Text can be tilted</div> would indeed pass this rule because it is not part of an instruction.